internal/bminventory/inventory_test.go

```diff
@@ -11389,134 +11389,6 @@ var _ = Describe("AMS subscriptions", func() {
 	})
 })
 
-var _ = Describe("[V2UpdateCluster] AMS subscriptions", func() {
```

Comment:
removed this, as the other copy is more up-to-date (additions from PRs such as https://github.com/openshift/assisted-service/pull/2546) and includes more tests.

see the diff here: https://www.diffchecker.com/YitBKZRp